app/src/main/java/com/cris/nvh/moviedb/ui/moviedetails/MovieDetailsActivity.java

```diff
@@ -0,0 +1,23 @@
+package com.cris.nvh.moviedb.ui.moviedetails;
+
+import android.content.Context;
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v7.app.AppCompatActivity;
+
+/**
+ * Created by nvh
+ * Contact: toiyeuthethao1997@gmail.com
+ */
+
+public class MovieDetailsActivity extends AppCompatActivity {
+    @Override
+    protected void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+    }
+
+    public static Intent getMovieDetailsActivityIntent(Context context) {
```

Comment:
Em có thể đổi tên cho gọn
```suggestion
    public static Intent getIntent(Context context) {
```